src/test/scala/karazin/scala/users/group/week1/homework/HomeworkSpecification.scala

```diff
@@ -23,14 +23,22 @@ object BooleanOperatorsSpecification extends Properties("Boolean Operators"):
   property("and") = forAll { (pair: (Boolean, Boolean)) =>
     val (left, right) = pair
     
-    and(left, right) == left && right
+    and(left, right) == (left && right)
   }
 
   property("or") = forAll { (pair: (Boolean, Boolean)) =>
     val (left, right) = pair
     
     or(left, right) == left || right
-  }   
+  }
+
+  property("eagerAnd") = forAll { (right: Boolean) =>
```

Comment:
Please use
```
propBoolean {
    [predicate]
}
```
instead of `forAll`, since you don't need any generated value.
The same for `or`.